code/modules/sanity/sanity_mob.dm

```diff
@@ -227,7 +236,8 @@
 
 /datum/sanity/proc/onSeeDeath(mob/M)
 	if(ishuman(M))
-		changeLevel(-SANITY_DAMAGE_DEATH(owner.stats.getStat(STAT_VIG)))
+		var/penalty = -SANITY_DAMAGE_DEATH(owner.stats.getStat(STAT_VIG))
+		changeLevel(penalty*death_view_multiplier)
```

Comment:
Creating a variable but not reusing more than 1 times elsewhere. Why ?

Reply:
you mean death_view_multiplier? the survivor perk modifies it

Reply:
I should have clarified, that's about the var/penalty. That's for readability ?

Reply:
Oh yeah, just for readability.  The expression would have been a bit too big otherwise. It's a simple local var anyway, it isn't a problem

Reply:
Oh yeah sorry, it's for readability. The expression would have been a pain to read otherwise. It's a local var anyway, it isn't an issue is it?

Reply:
Not much, if the byond compiler optimizes it away then that's fine

Reply:
even if it had a cost, it's negligible, don't worry